kolibri/content/models.py

```diff
@@ -18,13 +18,12 @@
 from le_utils.constants import content_kinds, file_formats, format_presets
 from le_utils.constants.languages import LANGUAGE_DIRECTIONS
 from mptt.models import MPTTModel, TreeForeignKey
-from mptt.querysets import TreeQuerySet
 
-from .content_db_router import get_active_content_database, get_content_database_connection
 from .utils import paths
 
 PRESET_LOOKUP = dict(format_presets.choices)
 
+CONTENT_SCHEMA_VERSION = '0.6.0'
```

Comment:
I think, as I am encoding the SCHEMA version here, we may as well make it independent of the Kolibri version.